src/Admin/AdminConsole/Controllers/ProvidersController.cs

```diff
@@ -275,4 +277,64 @@ public async Task<IActionResult> CreateOrganization(Guid providerId, Organizatio
 
         return RedirectToAction("Edit", "Providers", new { id = providerId });
     }
+
+    [HttpPost]
+    [SelfHosted(NotSelfHostedOnly = true)]
+    [RequirePermission(Permission.Provider_Edit)]
+    public async Task<IActionResult> Delete(Guid id, string providerName)
```

Comment:
When I try to delete a `Provider` in a `Pending` state from Admin, I get the following error:
![image](https://github.com/bitwarden/server/assets/144709477/046acebc-bc35-49bc-872b-515a3e82411a)

Reply:
this has been resolved